phpcs.xml.dist

```diff
@@ -273,4 +273,9 @@
         <!-- https://github.com/doctrine/orm/issues/8537 -->
         <exclude-pattern>src/QueryBuilder.php</exclude-pattern>
     </rule>
+
+     <rule ref="Generic.Files.LineEndings.InvalidEOLChar">
+      <exclude name="Generic.Files.LineEndings.InvalidEOLChar"/>
+     </rule>
```

Comment:
Why is this necessary?

Reply:
my bad, I'm on windows and when I make local test it is